### ISSUE TITLE
Validation warnings in admin not displayed or placed in wrong location.

### ISSUE THREAD

**Issue by mmzeeman** [zotonic/zotonic#1029]
Steps to reproduce:

When you place a single slash in de website field and close the address the enter url warning is placed in the wrong location or not placed at all.

This can completely confuse a user. (true story)

Is it possible to unhide the field before placing the warning?

![screen shot 2015-09-14 at 13 36 54](https://cloud.githubusercontent.com/assets/1024972/9849450/7bb35ac6-5aeb-11e5-8e9f-9c023c75ac0c.png)

**Comment by ArthurClemens** [comment]
This is a peculiar type of feedback that I haven't seen before. Shouldn't we use growl sticky messages, perhaps combined with feedback on the page near the save button?

**Comment by mworrell** [comment]
Yes, if there is no id then we should have maybe a kind of banner error close to the save button?

**Comment by mworrell** [comment]
At least we should open the *address* block, so it is clear that something is wrong there ....

**Comment by mmzeeman** [comment]
Exactly, it is a bit strange though that there is an invalid change in a block which is currently closed.

**Comment by ArthurClemens** [comment]
So possibly we also have a primary bug, insertion of character in a field.

**Comment by mmzeeman** [comment]
It could also be information which was inserted before the validator was in place, or that the user manually closed the block before hitting Save...

**Comment by mworrell** [comment]
Or autofill... won't be the first time an invisible field was automatically filled.

**Comment by ddeboer** [comment]
Is this still an issue?
